library/src/scala/quoted/Expr.scala

```diff
@@ -2,7 +2,7 @@ package scala.quoted
 
 import scala.runtime.quoted.Unpickler.Pickled
 
-sealed abstract class Expr[T] {
+sealed abstract class Expr[+T] {
```

Comment:
I think it'd make sense to make the subclasses covariant too for consistency.

Reply:
Yes, I will add apt them

Reply:
Done in #4896